administrator/components/com_finder/src/Controller/IndexerController.php

```
@@ -264,6 +282,7 @@ public static function sendResponse($data = null)
             $response->buffer = ob_get_contents();
             $response->memory = memory_get_usage(true);
         }
+        ob_clean();
```

looks like we try to silently hide errors, didn't you try to avoid this?